website/views.py

```diff
@@ -295,7 +295,11 @@ def resolve_guid(guid, suffix=None):
                     admin_group = group_helper.get_group('admin')
                     mod_group = group_helper.get_group('moderator')
                     # Check if user isn't a nonetype or that the user has admin/moderator permissions
-                    if auth.user is None or not (referent.node.has_permission(auth.user, permissions.ADMIN) or (mod_group.user_set.all() | admin_group.user_set.all()).filter(id=auth.user.id).exists()):
+                    if auth.user is None or not (
+                        referent.node.has_permission(auth.user, permissions.ADMIN) or
+                        (mod_group.user_set.all() | admin_group.user_set.all()).filter(id=auth.user.id).exists() or
```

Comment:
This line is probably superfluous; anyone in these groups should have the perm checked below.